website/docs/r/search_index.html.markdown

```diff
@@ -0,0 +1,340 @@
+
+
+### Analyzers
+An Atlas Search analyzer prepares a set of documents to be indexed by performing a series of operations to transform, filter, and group sequences of characters. You can define a custom analyzer to suit your specific indexing needs.
+
+* `name` - (Required) 	
+  Name of the custom analyzer. Names must be unique within an index, and may **not** start with any of the following strings:
+    * `lucene`
+    * `builtin`
+    * `mongodb`
+* `char_filters` - Array containing zero or more character filters. Always require a `type` field, and some take additional options as well
+  ```hcl
+  char_filters{
+   type = "<FILTER_TYPE>"
+   ADDITIONAL_OPTION = VALUE
+  }
+  ```
+  Atlas search supports four `types` of character filters:
+    * [htmlStrip](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-htmlStrip-ref) - Strips out HTML constructs
+        * `type` - (Required) Must be `htmlStrip`
+        * `ignored_tags`- a list of HTML tags to exclude from filtering
+        ```hcl
+          analyzers{
+            name = "analyzer_test"
+            char_filters={
+              type = "htmlStrip"
+              ignored_tags = ["a"]
+              }   
+            } 
+       ```
+    * [icuNormalize](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-icuNormalize-ref) - Normalizes text with the [ICU](http://site.icu-project.org/) Normalizer. Based on Lucene's [ICUNormalizer2CharFilter](https://lucene.apache.org/core/8_3_0/analyzers-icu/org/apache/lucene/analysis/icu/ICUNormalizer2CharFilter.html)
+    * [mapping](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-mapping-ref) - Applies user-specified normalization mappings to characters. Based on Lucene's [MappingCharFilter](https://lucene.apache.org/core/8_0_0/analyzers-common/org/apache/lucene/analysis/charfilter/MappingCharFilter.html)
+      An object containing a comma-separated list of mappings. A mapping indicates that one character or group of characters should be substituted for another, in the format `<original> : <replacement>`
+      >Note: this field needs to be in a JSON format using `EOF` tags
+      ### Example
+        ```hcl
+        analyzers{
+          type= "mapping"
+          mappings =<<-EOF
+          {
+             "\\" : "/"
+          }
+          EOF 
+      ```
+    * [persian](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-persian-ref) - Replaces instances of [zero-width non-joiners](https://en.wikipedia.org/wiki/Zero-width_non-joiner) with ordinary space. Based on Lucene's [PersianCharFilter](https://lucene.apache.org/core/8_0_0/analyzers-common/org/apache/lucene/analysis/fa/PersianCharFilter.html)
```

Comment:
I think this bullet is getting pulled in the above example.

Reply:
this bullet is at the same level as the previous char_filters